modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -58,6 +58,8 @@ def download_and_process
       begin
         @upload.update(metadata: @upload.metadata.merge(original_file_metadata(tempfile)))
 
+        validate_payload_size(tempfile)
```

Comment:
This occurs before the `VBADocuments::MultipartParser` is reached, so true "Empty payload" errors will be handled first.